src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxRandomBalls = 100;
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Ball ball = new Ball(colorSupplier.getRandomColor(), random.nextInt(maxRandomBalls));

```

Comment:
```suggestion
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
        Ball ball = new Ball(colorSupplier.getRandomColor(), random.nextInt(maxRandomBalls));

```

ColorSupplier should be also class level variable